CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 ## [Unreleased]
 
+## [2.0.0] - 2017-05-03
+### Added
+- Added new `tslint` rules from [v5.0.0](https://github.com/palantir/tslint/blob/master/CHANGELOG.md#v500) and [v5.1.0](https://github.com/palantir/tslint/blob/master/CHANGELOG.md#v510) releases
+    * `no-invalid-template-strings`
+    * `no-sparse-arrays`
+    * `no-unnecessary-callback-wrapper`
+    * `no-reference-import`
+    * `prefer-template`
+    * `return-undefined` (disabled)
+- Added new `tslint-react` rules from [v2.6.0](https://github.com/palantir/tslint-react/releases/tag/2.6.0) release [(#61)](https://github.com/Shopify/tslint-config-shopify/pull/61)
+    * `jsx-no-bind` (tslint-react)
```

Comment:
What does the `(tslint-react)` mean here?  That's already covered by the parent bullet point.